less/contact.less

```diff
@@ -0,0 +1,146 @@
+// Contact Content
+
+.contact {
+    padding-top: 5rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+
+    @media @phone {
+        padding-top: 15rem;
+    }
+
+    .main-contact-image img {
+        width: 60%;
+        height: auto;
+        margin-left: 15rem;
+        margin-bottom: 1rem;
+
+        @media @phone {
+            width: 70%;
+            margin-left: 5rem;
+        }
+    }
+
+    .main-contact-form {
+        
+        padding-top: 20rem;
+
+        .align-center {
+            text-align: center;
+         }
+        
+         // Grid
+        .row {
```

Comment:
interesting use of float. this is actually hard to learn. how long did it take you to figure this out?